site/content/en/docs/Installation/Install Agones/confirm.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Confirming Agones Installation"
+linkTitle: "Confirm Installation"
+weight: 40
+description: >
+  Confirming Agones is installed and started successfully.
+---
+
+To confirm Agones is up and running, run the following command:
+
+```bash
+kubectl describe --namespace agones-system pods
+```
+
+It should describe six pods created in the `agones-system` namespace, with no error messages or status. All `Conditions` sections should look like this:
+
+```
+Conditions:
+  Type              Status
+  Initialized       True
+  Ready             True
+  ContainersReady   True
+  PodScheduled      True
+```
+
+All this pods should be in a `RUNNING` state:
+
+
+```bash
+kubectl get pods --namespace agones-system
+
+NAME                                 READY   STATUS    RESTARTS   AGE
+agones-allocator-5c988b7b8d-cgtbs    1/1     Running   0          8m47s
+agones-allocator-5c988b7b8d-hhhr5    1/1     Running   0          8m47s
+agones-allocator-5c988b7b8d-pv577    1/1     Running   0          8m47s
+agones-controller-7db45966db-56l66   1/1     Running   0          8m44s
+agones-ping-84c64f6c9d-bdlzh         1/1     Running   0          8m37s
+agones-ping-84c64f6c9d-sjgzz         1/1     Running   0          8m47s
+```
+
+That's it! This creates the [Custom Resource Definitions][crds] that power Agones and allows us to define resources of type `GameServer`.
```

Comment:
nit: "This creates" doesn't make sense in this context. Maybe something like "You've now created CRDs that power agones and allow you to define resources of type GameServer".

Reply:
Agreed - this feels a bit weird. I took a stab at reworking it, PTAL.

If it's still not working, we can drop it in favour of something like "Agones is now installed! Congratulations!"